aten/src/ATen/native/cuda/BinaryMiscBackwardOpsKernels.cu

```diff
@@ -20,7 +20,7 @@ void sigmoid_backward_kernel_cuda(TensorIterator& iter) {
 }
 
 void tanh_backward_kernel_cuda(TensorIterator& iter) {
-  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "tanh_backward_cuda", [&]() {
+  AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES(iter.dtype(), "tanh_backward_cuda", [&]() {
```

Comment:
`AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES_AND1(ScalarType::Half, iter.dtype() ...)`

Reply:
Should be resolved here: https://github.com/pytorch/pytorch/pull/38486/commits/a38b4b34a2a70d9498ae25cddb3199805072449a. Thanks!